docker.sh

```diff
@@ -1,39 +1,55 @@
 #!/bin/bash
-set -eo pipefail
+set -euox pipefail
```

Comment:
-x will dump all the debug information which includes environmental properties like passwords. Is this safe to keep in?